store/vcs.go

```diff
@@ -177,7 +177,7 @@ func createVCS(ctx context.Context, tx *sql.Tx, create *api.VCSCreate) (*api.VCS
 	return &vcs, nil
 }
 
-func findVCSList(ctx context.Context, tx *sql.Tx, find *api.VCSFind) (_ []*api.VCS, err error) {
+func findVCSList(ctx context.Context, tx *sql.Tx, find *api.VCSFind) (_ []*api.VCSRaw, err error) {
```

Comment:
nit: not related to this change, we should not use named return values if possible.

([]*api.VCSRaw, error)

Reply:
There seems to be 32 occurrences of this pattern.
I will fix them in another PR.

Reply:
SG

Reply:
Fixed in #798